test/config/crashedtarget.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2019-present Open Networking Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"context"
+	"github.com/onosproject/helmit/pkg/helm"
+	"github.com/onosproject/onos-config/test/utils/proto"
+	gnmiapi "github.com/openconfig/gnmi/proto/gnmi"
+	"testing"
+	"time"
+
+	"github.com/onosproject/onos-api/go/onos/topo"
+
+	gnmiutils "github.com/onosproject/onos-config/test/utils/gnmi"
+)
+
+const (
```

Comment:
Because of go scoping rules, these constants and variables will end up in the namespace of every test. Putting these inside the test function is the best approach. If that's not feasible, then name these so they are unique for the test.